inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/SearchServiceImpl.java

```diff
@@ -66,11 +67,18 @@
     @Value(value = "${repository.path}")
     private String dir;
 
+    @Autowired
     public SearchServiceImpl()
     {
         indexes = new HashMap<>();
     }
 
+    public SearchServiceImpl(@Value("${data.path}") File aDir)
```

Comment:
The value annotation here is redundant with the Value annotation in line 67. The dir could be final. Anyway, we now have a class `RepositoryProperties` which can be injected as a dependency and which should be used instead.